chart/kubeapps/templates/NOTES.txt

```diff
@@ -6,7 +6,7 @@ Tip:
 
 Kubeapps can be accessed via port {{ .Values.frontend.service.port }} on the following DNS name from within your cluster:
 
-   {{ template "kubeapps.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+   {{ template "common.names.fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
```

Comment:
So, from now on, we shouldn't be worried about the length of the name anymore since this function is automatically truncating it, isn't it?

Reply:
Yup, it works pretty similar to what you had before.